SIPS/sip-7.md

```diff
@@ -84,13 +98,22 @@ interface HeadingComponent extends BaseComponent {
 
 interface TextComponent extends BaseComponent {
   type: 'text';
-  text: string;
+  value: string;
 }
 
 interface ButtonComponent extends BaseComponent {
   type: 'button';
   variant?: 'primary' | 'secondary';
-  text: string;
+  buttonType: 'button'
+  value: string;
+  name?: string;
+}
+
+interface SubmitButtonComponent extends BaseComponent {
+  type: 'button';
+  variant?: 'primary' | 'secondary';
```

Comment:
Do we need this for submit buttons? Shouldn't it just always be "primary"?

Reply:
People might want to have a secondary as submit and a primary for something else

Reply:
That seems like bad UX. 😅

Reply:
It's up to the snap developer IMO, it doesn't mean bad UX for all cases